.github/workflows/lint-renovate-pr.yml

```diff
@@ -0,0 +1,70 @@
+name: Lint and Update renovate PR
+
+# configure trigger by pull request
+on:
+  pull_request:
+    types: [opened, synchronize]
```

Comment:
Should we run this only when the helm files in `/operations/helm` have changed in the PR? Is this easy to detect?

Reply:
I initially thinking make this more general, like all potential updates after renovate PR, but let's try it in smaller scale just for helm first. will update.

Reply:
a larger scale is also ok by me. I just wasn't sure if that's what you intended. If we go with a larger scope, let's also include `reference-help` in the make targets